packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -17,9 +19,21 @@ const GlobalClassNames = {
   valueLabel: 'ms-Slider-value',
   showValue: 'ms-Slider-showValue',
   showTransitions: 'ms-Slider-showTransitions',
-  zeroTick: 'ms-Slider-zeroTick'
+  zeroTick: 'ms-Slider-zeroTick',
+  regularTick: 'ms-Slider-regularTick',
+  regularLabel: 'ms-Slider-regularLabel'
 };
 
+function getHighContrastSelector(systemColor: string): IRawStyle {
```

Comment:
I'm wondering if this should be put into the `utilities` package, seems like a very common scenario to encounter when styling.

Reply:
in that case it needs to be modified a little so it receives not a color string but the whole style object.